libs/unity/library/Runtime/Scripts/Media/MediaSender.cs

```diff
@@ -1,16 +1,20 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System.Threading.Tasks;
+using System;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.WebRTC.Unity
 {
     /// <summary>
-    /// Base class for media sources generating their media frames locally,
-    /// with the intention to send them to the remote peer.
+    /// Base class for media senders acting as automated bridges between a media track source
+    /// (frame producer) and a transceiver of a peer connection. The media sender is managed
+    /// by the peer connection automatically, so users typically do not use this class or one
+    /// of its derived classes directly.
     /// </summary>
-    public abstract class MediaSender : MediaSource
+    /// <seealso cref="AudioSender"/>
+    /// <seealso cref="VideoSender"/>
+    public abstract class MediaSender
```

Comment:
Any chance we can get rid of this and subclasses? They don't "send" anything, AFAICS they are boilerplate around a Track now. If we need them around maybe put into MediaLine (they are only used there) and/or make `internal`?

Reply:
First pass to see what it looks like : made `MediaSender` _et al._ `internal` for now, but it is still convenient to have a storage for all the properties of the sender track instead of munching them into the `MediaLine`. However looking more closely the only 2 properties of interest are the track name and the transceiver's codec preferences, so maybe they can move to the `MediaLine` and we move all logic code to maintain the local track into `MediaLine` too. Thoughts?

Reply:
Now removed completely in latest iteration.